tests/swo_rql/test_rql_base.py

```
@@ -285,20 +352,20 @@ def test_and_merge():
 @pytest.mark.parametrize("op", ["eq", "ne", "gt", "ge", "le", "lt"])
 def test_dotted_path_comp(op):
     rql_query = RQLQuery
-    assert str(getattr(rql_query().asset.id, op)("value")) == f"{op}(asset.id,value)"
-    assert str(getattr(rql_query().asset.id, op)(True)) == f"{op}(asset.id,true)"  # noqa: FBT003
-    assert str(getattr(rql_query().asset.id, op)(False)) == f"{op}(asset.id,false)"  # noqa: FBT003
-    assert str(getattr(rql_query().asset.id, op)(10)) == f"{op}(asset.id,10)"
-    assert str(getattr(rql_query().asset.id, op)(10.678937)) == f"{op}(asset.id,10.678937)"
+    assert str(getattr(rql_query().asset.id, op)("value")) == f"{op}(asset.id,'value')"
+    assert str(getattr(rql_query().asset.id, op)(True)) == f"{op}(asset.id,'true')"  # noqa: FBT003
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `FBT003`)

Remove unused `noqa` directive

(RUF100)